### PR TITLE
Improve error when knowledgebase is missing a description

### PR DESCRIPTION
**Pull request by collindutter**

### CODE REVIEW

File: griptape/tools/griptape_cloud_knowledge_base_client/tool.py [hunk 1]
```diff
@@ -52,4 +52,6 @@ def _get_knowledge_base_description(self) -> str:
             if "description" in response:
                 return response["description"]
             else:
-                raise ValueError(f'Error getting Knowledge Base description: {response["message"]}')
+                raise ValueError(
+                    "No description found for Knowledge Base. Please set a description, or manually set the `GriptapeCloudKnowledgeBaseClient.description` attribute."
```
Comment (vachillo) [inline griptape/tools/griptape_cloud_knowledge_base_client/tool.py:56]: maybe throw the id in the message?

**Review by vachillo — APPROVED** [review]

File: griptape/tools/griptape_cloud_knowledge_base_client/tool.py (hunk 1 above)
Comment (collindutter) [reply griptape/tools/griptape_cloud_knowledge_base_client/tool.py:56]: Does that come back in the response?

Comment (vachillo) [reply griptape/tools/griptape_cloud_knowledge_base_client/tool.py:56]: not sure, but cant you pull it from `self. knowledge_base_id`?

Comment (collindutter) [reply griptape/tools/griptape_cloud_knowledge_base_client/tool.py]: Oh duh

**Review by vachillo — APPROVED** [review]

**Pull request merged**